doc/source/conf.py

```diff
@@ -1,15 +1,19 @@
-from typing import Dict, Any
+from typing import Set, Dict, Any
 from datetime import datetime
 from importlib import import_module
 import os
 import sys
 from jinja2.filters import FILTERS
 import sphinx
 from sphinx.ext import autodoc
+from sphinx.ext.autosummary import generate
+from sphinx.util.inspect import safe_getattr
 from docutils import nodes
 import pathlib
 import logging
 
+DEFAULT_API_GROUP = "others"
```

Comment:
nit: `_DEFAULT_API_GROUP` ?

maybe use value `"_"` or "default" or "na" or "none"? "others" feel a bit weird as a default string.. or maybe it is commonly used in the docs world?

![image](https://github.com/ray-project/ray/assets/95255098/3f059fe4-6a67-4792-8b86-0861df6adb20)

Reply:
this word is rendered as a section name in the template (class_v2.rst) so i choose something readable